infracost-usage-example.yml

```diff
@@ -335,11 +335,12 @@ resource_usage:
     snapshot_export_size_gb: 200       # Size of snapshot that's exported to s3 in parquet format.
 
   aws_rds_cluster_instance.my_cluster:
-    monthly_cpu_credit_hrs: 24   # Number of hours in a month, where you expect to burst the baseline credit balance of a "t3" instance type.
-    vcpu_count: 2 # # (DEPRECATED this is now calculated automatically) Number of virtual CPUs allocated to your "t3" instance type. Currently instances with 2 vCPUs are available.
+    monthly_cpu_credit_hrs: 24                              # Number of hours in a month, where you expect to burst the baseline credit balance of a "t3" instance type.
+    vcpu_count: 2                                           # (DEPRECATED this is now calculated automatically) Number of virtual CPUs allocated to your "t3" instance type. Currently instances with 2 vCPUs are available.
     monthly_additional_performance_insights_requests: 10000 # Monthly Performance Insights API requests above the 1000000 requests included in the free tier.
     reserved_instance_term: 1_year                          # Term for Reserved Instances, can be: 1_year, 3_year.
     reserved_instance_payment_option: partial_upfront       # Payment option for Reserved Instances, can be: no_upfront (only for 1_year term), partial_upfront, all_upfront.
+    capacity_units_per_hr: 50                               # Number of Aurora Capacity Units per hour. Only used for Aurora Serverless V2 (when instance class is "db.serverless")
```

Comment:
we should also update instance-usage-defaults.*.yml and run `scripts/generateGlobalDefaultUsage.ts` in the dashboard.

Reply:
I've updated these. The problem is the minimum is 0.5 which costs $44 a month, so I've had to use that for all the sizes.